src/sys/socket/ffi.rs

```diff
@@ -33,9 +48,9 @@ pub struct msghdr<'a> {
     pub msg_name: *const c_void,
```

Comment:
Both `msghdr` and `cmsghdr` here should be removed in favor of the declarations in `libc`.